libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -159,8 +159,8 @@ class AC_AttitudeControl {
     void inertial_frame_reset();
 
     // Command a Quaternion attitude with feedforward and smoothing
-    // attitude_desired_quat: is updated on each time_step (_dt) by the integral of the angular velocity
-    virtual void input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_target);
+    // attitude_desired_quat: is updated on each time_step (_dt) by the integral of the body frame angular velocity
+    virtual void input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_body);
```

Comment:
should be:
 const Vector3f &ang_vel_body

Reply:
Done

Reply:
Changed it and realised that the input quaternion modifies the parameter on the stack. Happy to fix this on all the code I am responsible for in a single PR.